volatility3/framework/plugins/yarascan.py

```diff
@@ -61,19 +61,31 @@ class YaraScan(plugins.PluginInterface):
     """Scans kernel memory using yara rules (string or file)."""
 
     _required_framework_version = (2, 0, 0)
-    _version = (1, 1, 0)
+    _version = (1, 2, 0)
 
     # TODO: When the major version is bumped, take the opportunity to rename the yara_rules config to yara_string
     # or something that makes more sense
 
     @classmethod
     def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]:
-        return [
+        """Returns the requirements needed to run yarascan directly, combining the TranslationLayerRequirement
+        and the requirements from get_yarascan_option_requirements."""
+        return cls.get_yarascan_option_requirements() + [
```

Comment:
Does this handling affect the `windows.vadyarascan` plugin at all and/or should `vadyarascan` be updated to take a `ModuleRequirement` rather than a `TranslationLayerRequirement`?

Reply:
Great question! So currently `windows.vadyarascan` doesn't actually use the `yarascan` plugin at all. The windows one maintains it's own list of requirements to do the command line options, it does use the `yarascanner` scanner but not the plugin. 

If you think that this way of allowing the linux plugin to use the base config options from the `yarascan`  plugin then I would also look to update the windows to work in the same way. That way if you want to update the `yarascan` command line options there is only one place to do it.